README.md

```diff
@@ -2,6 +2,8 @@ Analytics client
 ================
 
 Client part of analytics services used at balena.
+The client talks to [analytics-backend](https://github.com/balena-io/analytics-backend) which processes the incoming events before forwarding them to the relevant parties, referred here as *the final destinations*.
+Currently the only final destination is Amplitude. We have plans to support more final destinations going forward, like Sentry.
```

Comment:
Ideally, as I see it, the final destinations should be handled in the `analytics-backend` as much as possible, and `analytics-client` should be just a friendly interface that doesn't have to worry about what happens next to ensure complete modularity.

In practice, it's probably next to impossible to avoid platform-specific (Mixpanel, Amplitude, Sentry) adaptations without reimplementing their SDKs - that's why we have these platforms mentioned in both `analytics-backend` and `analytics-client`.

Reply:
Agreed Pranas. Is there anything you suggest changing in this paragraph?
I like your sentence with modularity. We could paraphrase and add it here.

I'd like to mention Amplitude and Sentry to put concrete use case in the beginning of readme. So the component maintainer immediately gets what it means to integrate analytics-client. For sure, we should detail the specific of these final destinations on the analytics-backend docs.

Reply:
Maybe something along the following lines
```suggestion
The analytics client is a component that integrates into the [analytics-backend](https://github.com/balena-io/analytics-backend). Together they're responsible for processing the analytics tracking events before forwarding them to the relevant parties, referred to here as *the final destinations*.
Currently the only final destination is Amplitude. We have plans to support more final destinations going forward, like Sentry.
```

I don't think we should expand too much on the `analytics-backend` for the exact reasons of modularity and it being an independent component/repo :)

Reply:
Sounds good. Thanks Pranas.